CONTRIBUTING.md

```diff
@@ -245,27 +244,25 @@ Follow these guidelines:
 1. A footer can be added if this fixes a particular issue on GitHub.
 
 ```sh
-$ git add src/js/player.js
-$ git commit
+git add src/js/player.js
+git commit
 ```
 
-An example commit message:
-
-```
-fix: one line commit explanation
+An example of the first line of a commit message: `fix: changed the footer to correctly display foo`
 
 In the body of the commit message, we can talk about why we made the change. What the change entails.
+Any testing considerations or things to think about when looking at the commit. For Example:
 
-Any testing considerations or things to think about when looking at the commit.
-
-Fixes #123. The footer can contain Fixes messages.
+```txt
```

Comment:
I think this one got messed up in the automation. It supposed to look something like
```md
fix: one line commit explanation

In the body of the commit message, we can talk about why we made the change. What the change entails.

Any testing considerations or things to think about when looking at the commit.

Fixes #123. The footer can contain Fixes messages.
```